presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -261,7 +261,7 @@ public PlanOptimizers(Metadata metadata, SqlParser sqlParser, FeaturesConfig fea
                 new IterativeOptimizer(
                         stats,
```

Comment:
I think we need benchmarks for case of table with high number of partitions. See `BechmarkPlanner`